test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -66,35 +66,42 @@ public void parse_unknownCommandWord_returnsHelp() {
 
     @Test
     public void parse_helpCommand_parsedCorrectly() {
```

Comment:
The format doesn't correctly follow our coding standard as it does not mention that this method test the case-insensitive command. Should be `featureUnderTest_testScenario_expectedBehavior()`, e.g. `e.g. sortList_emptyList_exceptionThrown()`